tests/integration/test_keeper_disks/configs/enable_keeper.xml

```diff
@@ -48,7 +48,7 @@
         <raft_configuration>
             <server>
                 <id>1</id>
-                <hostname>node</hostname>
+                <hostname>node_logs</hostname>
```

Comment:
how did it work until now?

Reply:
It was simply ignoring this resolve error